mockredis/client.py

```diff
@@ -1354,6 +1356,27 @@ def _normalize_command_response(self, command, response):
 
         return response
 
+    # Config Set/Get commands #
+
+    def config_set(self, name, value):
+        """
+        Set a configuration parameter.
+        """
+        self.redis_config[name] = value
```

Comment:
You should be returning "OK". For more info, http://redis.io/commands/config-set